packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html

```diff
@@ -35,10 +35,14 @@
         data-translate="user_account_identity_documents_sub_title"
     ></p>
     <!--My Identity document description-->
-    <span
-        class="oui-tile__body"
-        data-translate="user_account_identity_documents_descriptiom"
-    ></span>
+    <p class="oui-tile__body">
+        <span
+            data-translate="{{'user_account_identity_documents_list_for_' + ($ctrl.user_type[$ctrl.currentUser] ? $ctrl.user_type[$ctrl.currentUser] : $ctrl.user_type['default'])}}"
```

Comment:
maybe can move the test $ctrl.user_type[$ctrl.currentUser] ? $ctrl.user_type[$ctrl.currentUser] : $ctrl.user_type['default'] in controller file ?

Reply:
I agress with @ghyenne 
```suggestion
            data-translate="{{:: 'user_account_identity_documents_list_for_' + $ctrl.getUserType()}}"
```
with one way binding

Reply:
done Guillaume